### ISSUE TITLE
Lock VID PID ID for the correct work of third-party software

### ISSUE THREAD

**Issue by PowerfulPony** [Valkirie/HandheldCompanion#962]
**Is your feature request related to a problem? Please describe.**
Every time the application is opened, the virtual controllers have different device IDs, which is why it breaks down with a specific device in a third-party application (as an example of reWASD - new controllers appear)

**Describe the solution you'd like**
I want to be able to remember the ID PID VID values. When creating a virtual controller, it must have the same identifiers with which it was created earlier.

**Describe alternatives you've considered**
Create ID PID VID once and do not change it

**Additional context**
At home I use the device connected to the TV with various controllers. To control them I use reWASD, each launch of HC creates a new controller in the list. And if the old one was configured, it forces you to configure it again every time you start HC.

**Comment by PowerfulPony** [comment]
ps/ when emulating a dual shock, the id does not change

**Comment by MSeys** [comment]
This is intended. The IDs are randomised for the 360 emulation to avoid certain issues.

**Comment by Valkirie** [comment]
> This is intended. The IDs are randomised for the 360 emulation to avoid certain issues. @Valkirie

Correct, I can check if it's indeed still required but I believe it is. Not sure how to circle this issue.

**Comment by CasperH2O** [comment]
Should this still happen if the Controller Manager is disabled @Valkirie ?

**Comment by Valkirie** [comment]
No

**Comment by CasperH2O** [comment]
@PowerfulPony can you try the virtual xbox 360 controller, with, on the Controller page, the Controller Manager disabled?

**Comment by PowerfulPony** [comment]
@CasperH2O I tried. result is the same. If you exit from HC and launch it again
